### DISCUSSION TITLE
Sideband F-Droid Repo

### DISCUSSION THREAD

**Discussion by faragher** [markqvist/Reticulum#474] [Show and tell]
For convenience, I've begun hosting an F-Droid repo for Sideband.

https://reticulum.betweentheborders.com/fdroid/repo/

Simply add the repo to your list, and you can install from F-Droid as well as keep up to date. Easy peasy.

Due mostly to not wanting to wade through fdroidserver's documentation and modifying metadata by hand, it only keeps the most recent release in the repo. Many other repos do the same, so why am I mentioning it? There's a second repo:

https://reticulum.betweentheborders.com/fdroid/archive

This currently hosts every version (except the latest) back to 0.7.7, I think. This will, drive space willing, store as many versions as practical. I highly recommend against using it for autoupdate, see "the server gets confused because it doesn't respect file dates, it cares about 'added to repo' date" above, I expect at least 5 - 10 old versions on the archive repo.

I may increase the main repo up to three as new versions are released, and may entirely depreciate the archive (unless we end up with LTS versions) but by and large, you can just download F-Droid, add the repo, and get an easy and updatable install.

**Comment by markqvist** [comment]
Thank you so much for setting this up!

**Comment by markqvist** [comment]
I've added the repo to the install section of the readme file and my own website as well. Thanks again!

**Comment by Erethon** [comment]
Added to the wiki as well, thank you!

**Reply by markqvist** [reply]
Thanks!
